### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by deining**

### CODE REVIEW

File: src/pages/roadmap.mdx
Comment (jameslittle230) [inline src/pages/roadmap.mdx:25]: This one actually isn't a typo! I'm describing [a single search index split up into different files](https://en.wikipedia.org/wiki/Shard_(database_architecture)) here.

**Review by jameslittle230 — CHANGES_REQUESTED** [review]
Thanks for catching these!

One revert request - once you update I'm happy to approve and let you merge.

File: src/pages/roadmap.mdx
Comment (deining) [reply src/pages/roadmap.mdx]: Thanks for clarification! I reverted this meanwhile.

**Review by jameslittle230 — APPROVED** [review]
Thanks very much! Feel free to merge whenever you want

**Comment by deining** [comment]
> Thanks very much!

You are welcome.

> Feel free to merge whenever you want

I don't have write access to the repo, so I guess you have to merge my PR.

**Pull request merged**
